package.json

```diff
@@ -21,7 +21,7 @@
     "fs-extra": "^7.0.0",
     "glob": "^7.1.1",
     "husky": "1.1.1",
-    "jest": "^23.5.0",
+    "jest": "^24.0.0",
     "jest-cli": "^23.5.0",
```

Comment:
Should also update `babel-jest`, fixes #10913.

Reply:
good call; thanks!

Reply:
Yeah - this seems to break the [Windows pipeline](https://dev.azure.com/gatsbyjs/gatsby/_build/results?buildId=5081) 😩 

> TypeError: Cannot read property 'cwd' of undefined

@stefanprobst is there a substantive reason we should make this change?

Reply:
I remember having a problem with the version mismatch between `babel-jest@22` and `jest@23` in current master at some point, that's where the PR came from. Does updating `jest-cli` help?

Reply:
And the snapshot changes seem to be because of https://github.com/facebook/jest/pull/6381